src/dashboard/src/components/sheet/Sheet.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from './Sheet.module.scss';
+
+export interface ISheetProps {
+  children?: React.ReactNode;
+}
+
+export const Sheet = ({ children }: ISheetProps) => {
```

Comment:
Basic component that can be used to style the default page design (white background with shadow).

![image](https://github.com/bcgov/hsb-dashboard/assets/3180256/ad6e549e-fb68-488e-8cdf-2d1150969b1e)